releases.md

```diff
@@ -47,6 +47,7 @@ or eigenvectors.
 - drop_missing : helper function analogous to statsmodels missing='drop' mechanism which allows the user to evict 
 features and observations where one or more data points is not finite such that OLS may then be performed on dense / 
 complete data.
+- EWMA functions for 1D and 2D data sets (dense data / no NaNs)
```

Comment:
My bad - this will need moving to a 2019.2 section, but happy to do that in a later PR

Reply:
I think it's fine - the original EWMA function made the cut and it handles 1D and 2D dense data.  The addition in this PR is just a faster 2D-only version really.

Reply:
:thumbsup: nice one, will merge. we can add a `speedup for 2d ewma` enhancement message for 2019.2 later.